test/integration/aab_offline_test.go

```diff
@@ -49,6 +49,40 @@ func TestOffline(t *testing.T) {
 				// RFC1918 address that unlikely to host working a proxy server
 				env = append(env, "HTTP_PROXY=172.16.1.1:1")
 				env = append(env, "HTTP_PROXYS=172.16.1.1:1")
+
+				c.Env = env
+				rr, err := Run(t, c)
+				if err != nil {
+					// Fatal so that we may collect logs before stop/delete steps
+					t.Fatalf("%s failed: %v", rr.Command(), err)
+				}
+			})
+		}
+	})
+}
+
+func TestOfflineDownload(t *testing.T) {
```

Comment:
we might not need to do this , since we run download only before all tests

Reply:
removed